src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DeclarationOrderCheck.java

```diff
@@ -84,41 +84,112 @@
  * </li>
  * </ul>
  * <p>
- * To configure the check:
+ * To configure the check with default config:
  * </p>
  * <pre>
  * &lt;module name=&quot;DeclarationOrder&quot;/&gt;
  * </pre>
- * <p>
- * With default options:
- * </p>
+ * <p>Example:</p>
  * <pre>
- * class K {
- *   int a;
- *   void m(){}
- *   K(){}  &lt;-- &quot;Constructor definition in wrong order&quot;
- *   int b; &lt;-- &quot;Instance variable definition in wrong order&quot;
+ * public class Test {
+ *
+ *   public int a;
+ *   protected int b;
+ *   public int c;            // violation, variable access definition in wrong order
+ *
+ *   Test() {
+ *     this.a = 0;
+ *     this.b = 0;
+ *     this.c = 0;
+ *     this.name = "";
+ *   }
+ *
+ *   public void foo() {
+ *     // This method does nothing
+ *   }
+ *
+ *   Test(int a, int b, int c, String name) {   // violation, constructor definition in wrong order
+ *     this.a = a;
+ *     this.b = b;
+ *     this.c = c;
+ *     this.name = name;
+ *   }
+ *
+ *   private String name;     // violation, instance variable declaration in wrong order
  * }
  * </pre>
  * <p>
- * With <b>ignoreConstructors</b> option:
+ * To configure the check to ignore validation of constructors:
  * </p>
  * <pre>
- * class K {
- *   int a;
- *   void m(){}
- *   K(){}
- *   int b; &lt;-- &quot;Instance variable definition in wrong order&quot;
+ * &lt;module name=&quot;DeclarationOrder&quot;&gt;
+ *   &lt;property name=&quot;ignoreConstructors&quot; value=&quot;true&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *
+ *   public int a;
+ *   protected int b;
+ *   public int c;            // violation, variable access definition in wrong order
+ *
+ *   Test() {
+ *     this.a = 0;
+ *     this.b = 0;
+ *     this.c = 0;
+ *     this.name = "";
+ *   }
+ *
+ *   public void foo() {
+ *     // This method does nothing
+ *   }
+ *
+ *   Test(int a, int b, int c, String name) {   // OK, validation of constructors ignored
+ *     this.a = a;
+ *     this.b = b;
+ *     this.c = c;
+ *     this.name = name;
+ *   }
+ *
+ *   private String name;     // violation, instance variable declaration in wrong order
  * }
  * </pre>
  * <p>
- * With <b>ignoreConstructors</b> option and without a method definition in a source class:
+ * To configure the check to ignore modifiers while validating:
```

Comment:
` while validating` I think it is pretty clear without it

Reply:
Done.